openedx_external_enrollments/models.py

```diff
@@ -38,3 +40,26 @@ class Meta(object):
         Model meta class.
         """
         app_label = "openedx_external_enrollments"
+
+
+class OtherCourseSettings(models.Model):
+    """
+    Model to persist other course settings.
+    """
+
+    course_overview = models.OneToOneField(
+        CourseOverview,
+        on_delete=models.CASCADE,
+        primary_key=True,
+    )
+    external_course_id = models.CharField(max_length=50, null=False, blank=True)
+    external_platform = models.CharField(max_length=20, null=False, blank=True)
```

Comment:
These max_lenghts must be in accordance with the values of the edx platform. @ivanvgh

Reply:
The external_course_id field is left as the platform and external_platform left as 255 as max length as well.